schema.ts

```diff
@@ -178,6 +178,12 @@ export enum ValueHintType {
    * Indicates to render a value as a select list.
    */
   SelectList = 'selectList',
+
+  /**
+   * Indicates an external user id
+   * @hidden
+   */
+  UserId = 'userId',
```

Comment:
Should this be called ExternalUserId or something more specific depending on the answer to the comment above?

Reply:
And if so, I'm not sure how much that applies to.